geopandas/array.py

```diff
@@ -1806,6 +1807,12 @@ def convert_values(param):
     def __eq__(self, other):
         return self._binop(other, operator.eq)
 
+    # https://github.com/python/typeshed/issues/2148#issuecomment-520783318
+    # Incompatible types in assignment (expression has type "None", base class
+    # "object" defined the type as "Callable[[object], int]")
+    # (Explicitly mirrored from pandas to declare non hashable)
+    __hash__: ClassVar[None]  # type: ignore[assignment]
```

Comment:
this is to solve geopandas\array.py:315:7: PLW1641 Object does not implement `__hash__` method

we could just ignore that rule but I find ignores on class definitions awkward